src/resolversFactory.js

```diff
@@ -1,7 +1,18 @@
 var util = require('util');
 var http = require('http');
+var peliasConfig = require( 'pelias-config' ).generate();
+
+
+function createWofPipResolver(url, config) {
+  config = config || peliasConfig;
+
+  var maxConcurrentReqs = 1;
+  if (config.imports.adminLookup && config.imports.adminLookup.maxConcurrentReqs) {
+    maxConcurrentReqs = config.imports.adminLookup.maxConcurrentReqs;
+  }
+
+  http.globalAgent.maxSockets = maxConcurrentReqs;
```

Comment:
It's possible to create an [HTTP Agent](https://nodejs.org/docs/latest-v0.12.x/api/http.html#http_class_http_agent) as a local variable, set the max sockets on there, and then pass that agent to the request module via the [options parameter](https://github.com/request/request#requestoptions-callback).

We should probably do that as it will let us independently set the max number of sockets in different places. I can imagine us wanting to set the maxSockets to a different value if we also wrap the address deduplicator service in parallel-stream (which we definitely should do).